application/controllers/TodoController.php

```diff
@@ -5,13 +5,35 @@ class TodoController extends MY_Controller {
 
 	public function __construct() {
 		parent::__construct();
-		// load model here
+			$this->load->database();
```

Comment:
I think this is unnecessary because you already loaded the database library in autoload file. You can just load the model/s directly.